app/javascript/bundles/UI/components/Alert/Alert.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import css from './Alert.scss';
+import classnames from 'classnames';
+
+var types = ['standard', 'sky', 'fire', 'sun', 'sea'];
+
+
+const Alert = ({type, content}) => {
```

Comment:
I would change `content` for `children` so you can use this componente like this:

```jsx
<Alert type="sky">My content</Alert>
```